PORTAL/jobs/_pyperformance.py

```diff
@@ -647,16 +745,20 @@ def from_raw(cls, raw, *, fail=None):
             return None
 
     @classmethod
-    def _parse_value(cls, valuestr):
+    def _parse_value(cls, valuestr: str) -> _utils.ElapsedTimeWithUnits:
```

Comment:
Note that the return type differs for subclasses.  This has a cascade effect that impacts various types in subclasses (e.g. the return type of the `byname` property.  It isn't clear to me how to resolve that for mypy.  At the least, I'll note where it matters.

Reply:
Variance on the return type isn't well-supported.  My solution to this was to use `Any` as the return type for `_parse_value`, and then adding an assert that it's a `PyperfComparisonValue` in `PyperfComparison.look_up`, which basically turns it into a runtime check, but probably good enough for this corner case for now.